public/app/core/components/AppChrome/SectionNav/SectionNavItem.tsx

```diff
@@ -22,7 +23,7 @@ export function SectionNavItem({ item, isSectionRoot = false }: Props) {
   const linkClass = cx({
     [styles.link]: true,
     [styles.activeStyle]: item.active,
-    [styles.isSection]: Boolean(children?.length) || item.isSection,
+    [styles.isSection]: level < 2 && (Boolean(children?.length) || item.isSection),
```

Comment:
Wonder if the level number should be moved to a constant to keep it in one place?

Reply:
extracted it into a constant and also applied the same logic in `DockedMegaMenu` (it has the same problem)